README.md

```diff
@@ -27,57 +28,59 @@ released copy of these modules.
-		    <configuration>
-		        <artifactItems>
-		            <artifactItem>
-		                <groupId>org.wildfly</groupId>
-		                <artifactId>wildfly-dist</artifactId>
-		                <version>${wildfly.version}</version>
-		                <type>zip</type>
-		                <overWrite>true</overWrite>
-		                <outputDirectory>${project.build.directory}/node1</outputDirectory>
-		            </artifactItem>
-		            <artifactItem>
-		                <groupId>org.hibernate.lucene-jbossmodules</groupId>
-		                <artifactId>lucene-jbossmodules</artifactId>
-		                <version>${lucene-modules.version}</version>
-		                <type>zip</type>
-		                <classifier>dist</classifier>
-		                <overWrite>true</overWrite>
-		                <outputDirectory>${project.build.directory}/node1/wildfly-${wildfly.version}/modules</outputDirectory>
-		            </artifactItem>
-		        </artifactItems>
-		    </configuration>
-		</execution>
-	    </executions>
+## Usage: server provisioning via Maven
+
+Maven users can use the `wildfly-server-provisioning-maven-plugin` to create a custom WildFly server including these modules:
+
+	<plugins>
+		<plugin>
+		<groupId>org.wildfly.build</groupId>
+		<artifactId>wildfly-server-provisioning-maven-plugin</artifactId>
+		<version>1.2.0.Final</version>
+		<executions>
+			<execution>
+			<id>server-provisioning</id>
+			<goals>
+				<goal>build</goal>
+			</goals>
+			<phase>compile</phase>
+			<configuration>
+				<config-file>server-provisioning.xml</config-file>
+				<server-name>minimal-wildfly-with-lucene</server-name>
+			</configuration>
+			</execution>
+		</executions>
 	</plugin>
 
-This will make them available as an opt-in dependency to any application deployed on WildFly.
+You will also need a `server-provisioning.xml` in the root of your project:
+
+	<server-provisioning xmlns="urn:wildfly:server-provisioning:1.1">
+		<feature-packs>
+	
+			<feature-pack
+				groupId="org.hibernate.lucene-jbossmodules"
+				artifactId="lucene-jbossmodules"
+				version="${lucene-modules.version}"/>
+	
+			<feature-pack
+				groupId="org.wildfly"
+				artifactId="wildfly-servlet-feature-pack"
+				version="${your-preferred-wildfly.version}" />
+	
+		</feature-packs>
+	</server-provisioning>
+
+This will make Lucene available as an opt-in dependency to any application deployed on WildFly.
 To enable the dependency there are various options, documented in [Class Loading in WildFly](https://docs.jboss.org/author/display/WFLY/Class+Loading+in+WildFly).
 
+N.B. the current version of these modules has been tested with `WildFly 11.0.0.Final`.
+
+## Non-Maven users
+
+Plugins for other build tools have not been implemented yet, but this should be quite straight forward to do: the above Maven plugin is just a thin wrapper invoking into other libraries; these other libraries are build agnostic and are responsible for performing most of the work.
```

Comment:
"invoking into other libraries"? Is this correct? I would have said "invoking other libraries"...

Reply:
I think into is clearer, but since it confuses you I'm removing it.